code/game/objects/items/weapons/twohanded.dm

```diff
@@ -272,3 +272,36 @@
 
 	force = 16
 	throwforce = 25
+
+/obj/item/binoculars
+	name = "binoculars"
+	desc = "Used for long-distance surveillance."
+	icon_state = "binoculars"
+	item_state = "syringe_kit"
```

Comment:
I could make a different in-hand sprite for them holding the binoculars up to their face, but why is it necessary?
Why would other people need to know that the player is looking through binoculars?